README.md

```diff
@@ -167,55 +198,53 @@ Back in your local text editor in `index.html`, add the following code to your h
 1. Import the Ethersjs source into your `index.html` page inside a new set of script tags:
 
 ```html
-<script charset="utf-8"
-        src="https://cdn.ethers.io/scripts/ethers-v4.min.js"
-        type="text/javascript">
- </script>
+<script
+  charset="utf-8"
+  src="https://cdn.ethers.io/scripts/ethers-v4.min.js"
+  type="text/javascript"
+></script>
 
 <script>
   ////////////////////
   //ADD YOUR CODE HERE
   ////////////////////
 </script>
-
 ```
 
-2. Inside a new script tag, ensure ethereum is enabled:
+2. Inside a new script tag define an ethers provider. In our case it is Ropsten:
 
 ```javascript
-window.ethereum.enable();
+var provider = new ethers.providers.Web3Provider(window.provider, "ropsten");
 ```
 
-3. Now, define an ethers provider. In our case it is Ropsten:
+3. Request access to the user's wallet
 
 ```javascript
-var provider = new ethers.providers.Web3Provider(
-  web3.currentProvider,
-  "ropsten"
-);
+(async () => await provider.send("eth_requestAccounts", []))();
```

Comment:
Do you mind changing this to use `.then()` instead of anonymous async-await so as to make it easier for beginners to understand.

Reply:
I can do that but I could also just change the script tag to be `<script type="module">` so top-level await is able to be used, which would make it easier to understand, while also being more up-to-date JavaScript. I'm not good at deciding which is better for newer developers, what do you recommend?

Reply:
@haardikk21 check my latest commits, I made it use `.then` :)

Reply:
i prefer async/await, but my sole concern here is with the anonymous function. having to do `(async () => await ...)()` is weird unless you understand it, thats why i suggested `.then`

thank you!

Reply:
please let me know your Discord username so I can give you your role

Reply:
jords#0763